examples/installation_on_windows/README.md

```diff
@@ -0,0 +1,109 @@
+# Install ServerlessLLM on Windows Machines
+
+To install the ServerlessLLM dependencies, Ubuntu 20.04 is required according to the installation guide. Some packages, such as `vllm`, cannot be installed natively on Windows. Therefore, a Linux environment is necessary. Windows Subsystem for Linux (WSL) provides a convenient way to run Linux on Windows machines.
+
+## 1 Windows Subsystem for Linux
+It is recommended to download the Ubuntu app from the Microsoft Store. This enables a fully functional Linux terminal environment.
+
+Advantages:
+Allows installation of all required dependencies.
+Seamlessly integrates with Windows.
+
+Limitations:
+WSL does not support GUI applications by default.
+Configuring variable paths can occasionally be challenging.
+
+You can install Ubuntu from the Microsoft Store.
+
+![alt text](Ubuntu-app.png)
+
+**Description**:
+Install a complete Ubuntu terminal environment in minutes using WSL. This allows you to:
+- Develop cross-platform applications.
+- Enhance workflows for data science or web development.
+- Manage IT infrastructure—all without leaving Windows.
+
+For more details on Ubuntu WSL and Canonical's developer support, visit: [https://ubuntu.com/wsl](https://ubuntu.com/wsl).
+
+### 1.2 Setting Up WSL
+For a step-by-step video guide, check out this [YouTube tutorial](https://www.youtube.com/watch?v=sUsTQTJFmjs) by ProgrammingKnowledge. It covers the process in detail, including every button click.
+
+If the installation is successful, your terminal should look like this:
+
+<img src="wsl-terminal.png" alt="WSL Terminal" width="800"/>
+
+---
+
+## 2. Configuring WSL for ServerlessLLM
+
+### 2.1 Checking Memory Status
+
+If your configuration does not meet the requirements below, you need to update the `.wslconfig` file located in C:\Users\your_username\\. If this file does not exist, create it manually.
+You can check the current memory status using:
+
+```bash
+free -h
+```
+
+The minimum setting to start `sllm-store-server` is
+```bash
+[wsl2]               # this line is a must-have
+memory=5GB           # Limits VM memory
+swap=2GB             # Sets swap file size
+```
+However, depending on the size of models you want to deploy, you need to increase the configuration accordingly in most cases. For example, the WSL environment must meet the following minimum memory requirements to deploy the sample model. Using the Hugging Face backend can avoid the issue with passing the dtype parameter to the vLLM model downloader. Also, it allows a much faster deployment. Therefore, it is recommended to use the command below to deploy the `facebook/opt-13.b` model.
```

Comment:
Hi, there’s a typo—it should be opt-1.3b. The dtype issue has been resolved, so please exclude it. Based on your tests, deploying an opt-1.3b model requires 16GB of memory and 8GB of swap, correct? Is memory=5GB, swap=2GB not able to run this model?